<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiMartingale</class>
 <widget class="QWidget" name="GuiMartingale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GuiMartingale</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QGroupBox" name="settingsBox">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="labelPeriod">
        <property name="text">
         <string>Period (minutes)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="labelPair">
        <property name="text">
         <string>Pair</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" colspan="3">
       <widget class="QComboBox" name="pair">
        <item>
         <property name="text">
          <string>BTCUSD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ETHUSD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LTCUSD</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="3" colspan="3">
       <widget class="QDoubleSpinBox" name="startVolume">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>9999999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QCheckBox" name="repeatProfit">
        <property name="text">
         <string>repeat after profit</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="labelStartVolume">
        <property name="text">
         <string>Start volume</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="4">
       <widget class="QCheckBox" name="stopAndSale">
        <property name="text">
         <string>stop and sale on failure</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="labelBid">
        <property name="text">
         <string>Buy price</string>
        </property>
       </widget>
      </item>
      <item row="8" column="5">
       <widget class="QDoubleSpinBox" name="ask">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>9999999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="labelAsk">
        <property name="text">
         <string>Sell price</string>
        </property>
       </widget>
      </item>
      <item row="7" column="5">
       <widget class="QDoubleSpinBox" name="bid">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>9999999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="5">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Stop loss</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QSplitter" name="splitter_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Value</string>
            </property>
           </widget>
           <widget class="QDoubleSpinBox" name="stopLossValue">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="maximum">
             <double>9999999.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSplitter" name="splitter">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Percent</string>
            </property>
           </widget>
           <widget class="QDoubleSpinBox" name="stopLossPercent">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelExchange">
        <property name="text">
         <string>Exchange</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="3">
       <widget class="QComboBox" name="exchange">
        <item>
         <property name="text">
          <string>Bitfinex</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="3" colspan="3">
       <widget class="QSpinBox" name="period">
        <property name="maximum">
         <number>9999</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="9" column="1" colspan="5">
       <widget class="QPushButton" name="start">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1" colspan="5">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSplitter" name="splitter_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="labelLog">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Log</string>
      </property>
     </widget>
     <widget class="QListWidget" name="log"/>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="statusBox">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Status</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="labelPrice">
        <property name="text">
         <string>Average price</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="cost">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="maximum">
         <double>9999999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="price">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="maximum">
         <double>9999999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelVolume">
        <property name="text">
         <string>Volume</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="volume">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>9999999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelCost">
        <property name="text">
         <string>Cost</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="horizontalSpacing">
         <number>20</number>
        </property>
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="stop">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="sell">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Sell</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
